internal/kokoro/continuous.sh

```diff
@@ -34,6 +34,7 @@ export GCLOUD_TESTS_GOLANG_KEY=$GOOGLE_APPLICATION_CREDENTIALS
 export GCLOUD_TESTS_GOLANG_DATASTORE_DATABASES=database-01
 export GCLOUD_TESTS_GOLANG_FIRESTORE_PROJECT_ID=gcloud-golang-firestore-tests
 export GCLOUD_TESTS_GOLANG_FIRESTORE_KEY=$KOKORO_KEYSTORE_DIR/72523_go_firestore_integration_service_account
+export GCLOUD_TESTS_GOLANG_FIRESTORE_DATABASES=(default),database-02
```

Comment:
Similar to datastore, let's only require *additional* databases to be specified in this env variable (default should always run, even if this isn't set)

Reply:
Added